takwimu/templates/takwimu/profile_page.html

```diff
@@ -22,4 +22,29 @@
 
   {% include 'takwimu/_includes/sections/updates.html' %}
 
+  <script>
+    document.addEventListener('DOMContentLoaded', function () {
+      // Sync selected tabs to reflect on the indicator pagination
+      // Handling multiple IDS
+      $(".data-topic-container").each(function () {
+        var topic_id = this.id;
+        var nav_tabs = "#" + topic_id + " .dataindicators-nav .nav-link, #" + topic_id +
+          " .indicator-pagination .nav-link";
+        //Set all nav tabs to inactive
+        $(nav_tabs).click(function () {
+          //get selected href
+          var href = $(this).attr('href');
+          $(nav_tabs).removeClass("active").attr("aria-selected", false);
+          //get all nav tabs matching the href and set to active
+          var matching_navtabs = "#" + topic_id + " .dataindicators-nav a[href='" + href + "'], #" + topic_id +
+            "\ .indicator-pagination a[href='" + href + "']";
+
+          $(matching_navtabs).addClass("active").attr("aria-selected", true);
+          //Trigger active tab
+          $(".topic-" + topic_id + " .tab-pane").removeClass("active show");
+          $(".topic-" + topic_id + " .tab-pane" + href).addClass("active show");
+        });
+      });
+    });
```

Comment:
@DavidLemayian 
To sync sidebar tabs and dot tabs underneath the indicator, i had to use that technique, 
Other wise if i use two different tabs on the same content, It will work but it wont be in sync

Reply:
@mtuchi I'm saying in terms of the event listener - implementation should follow the link I shared with you.